loader/src/main/java/eu/europeana/fulltext/loader/service/XMLParserService.java

```diff
@@ -56,30 +58,36 @@
     private static final String RDF_NAMESPACE = "http://www.w3.org/1999/02/22-rdf-syntax-ns#";
     private static final String RDF = "RDF";
 
-    private static final String FULLTEXTRESOURCE = "FullTextResource";
-    private static final String FULLTEXTRESOURCE_ABOUT = "about";
-    private static final String FULLTEXTRESOURCE_VALUE = "value";
-    private static final String FULLTEXTRESOURCE_LANGUAGE = "language";
+    private static final String FULLTEXTRESOURCE            = "FullTextResource";
+    private static final String FULLTEXTRESOURCE_ABOUT      = "about";
+    private static final String FULLTEXTRESOURCE_VALUE      = "value";
+    private static final String FULLTEXTRESOURCE_LANGUAGE   = "language";
 
-    private static final String ANNOTATION = "Annotation";
-    private static final String ANNOTATION_ID = "ID";
+    private static final String ANNOTATION                  = "Annotation";
+    private static final String ANNOTATION_ID               = "ID";
 
-    private static final String ANNOTATION_TYPE = "type";
-    private static final char   ANNOTATION_TYPE_PAGE = 'P';
+    private static final String ANNOTATION_TYPE             = "type";
+    private static final char   ANNOTATION_TYPE_PAGE        = 'P';
```

Comment:
Since we also define P, M and C in the API module, perhaps it's better to move this to commons module

Reply:
I forgot to remove these ... they are now gone.